test/test_vim.py

```diff
@@ -22,7 +22,8 @@ def test_command():
     vim.command('w')
     ok(os.path.isfile(fname))
     eq(open(fname).read(), 'testing\npython\napi\n')
-    os.unlink(fname)
+    if os.name != 'nt':
```

Comment:
No amount of sleeping seems to have any affect..still results in error:

`PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\brcolow\\AppData\\Local\\Temp\\tmp9zm0bm83'`